pkg/controller/backup_controller.go

```diff
@@ -392,6 +393,11 @@ func (controller *backupController) runBackup(backup *api.Backup, bucket string)
 	backupScheduleName := backup.GetLabels()["ark-schedule"]
 	controller.metrics.SetBackupTarballSizeBytesGauge(backupScheduleName, backupSizeBytes)
 
+	backup.Status.CompletionTimestamp.Time = controller.clock.Now()
+	backupDuration := backup.Status.CompletionTimestamp.Time.Sub(backup.Status.StartTimestamp.Time)
+	backupDurationSeconds := float64(backupDuration / time.Second)
```

Comment:
why's the ` / time.Second` needed? (Not saying it's wrong, just wondering)

Reply:
Good catch - that should be multiplied to get the seconds, since Duration is nanoseconds.

Reply:
Actually I'm confusing myself here - this is correct. We need to divide to get the seconds out of a Duration.

Multiplication would be seconds to nanoseconds.

Reply:
To elaborate a bit more: the histogram we're building uses seconds as it's unit, which is why we're going from ns to seconds.

Reply:
got it, ok. duration math is weird :)